proposals/0204-same-app-from-multiple-devices.md

```diff
@@ -43,14 +43,40 @@ There is no API change. Just updates some descriptions.
 +        Needs to be unique over all applications from the same device.
         May not be empty.
         May not start with a new line character.
-        May not interfere with any name or synonym of previously registered applications and any predefined blacklist of words (global commands)
+-        May not interfere with any name or synonym of previously registered applications and any predefined blacklist of words (global commands)
++        May not interfere with any name or synonym of previously registered applications from the same device and any predefined blacklist of words (global commands)
 -        Needs to be unique over all applications. Applications with the same name will be rejected.
 +        Additional applications with the same name from the same device will be rejected.
        Only characters from char set [@TODO: Create char set (character/hex value) for each ACM and refer to] are supported.
     </description>
 </param>
 ```
 
+```xml
+<param name="ttsName" type="TTSChunk" minsize="1" maxsize="100" array="true" mandatory="false" since="2.0">
+    <description>
+        TTS string for VR recognition of the mobile application name, e.g. "Ford Drive Green".
```

Comment:
Please remove Ford specific references from API.

Reply:
The string has been already in the xml API file. I didn't introduce it in the proposal.